test/karma.conf.js

```diff
@@ -10,6 +10,8 @@ module.exports = function(config) {
     files: [
       'lib/jquery/dist/jquery.js',
       'lib/bootstrap-datepicker/dist/js/bootstrap-datepicker.js',
+      'lib/moment/moment.js',
+      'lib/eonasdan-bootstrap-datetimepicker/src/js/bootstrap-datetimepicker.js',
```

Comment:
Yes, I was kind of waiting to see the results of #337 before commenting on this. Since that doesn't seem to be moving too quickly, we could add this and I can adjust that PR to move this directive into the jquery based module.

Just be advised that the module will likely change and need to be specified in the application. See #337 for details.